docs/conf.py

```diff
@@ -273,6 +273,11 @@
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 # texinfo_show_urls = 'footnote'
 
+# Ignore link checks for github issue urls
```

Comment:
```suggestion
# Avoid errors due to GitHub rate limit
```

Reply:
I have updated the text in the latest commit.